src/services/API/ChatsAPI.ts

```diff
@@ -0,0 +1,82 @@
+import BaseAPI from "./BaseAPI";
+
+export interface GetChatData {
+  offset: number;
+  limit: number;
+  title: string;
+}
+
+export interface CreateChatData {
+  title: string;
+}
+export interface GetUsersChatData {
+  id: number;
+  params: {
+    offset: number;
+    limit: number;
+    name: string;
+    email: string;
+  };
+}
+export interface UploadChatAvatarData {
+  chatId: number;
+  avatar: FormData;
+}
+
+export interface AddUsersData {
+  chatId: number;
+  users: number[];
+}
+
+export class ChatsAPI extends BaseAPI {
+  constructor() {
+    super("/chats");
+  }
+
+  getChats(data: GetChatData): Promise<any> {
+    return this.http.get("/", { data, isQueryParams: true });
+  }
+
+  createChat(data: CreateChatData): Promise<any> {
+    return this.http.post("/", { data }).catch((e) => {
+      console.log(e, "catch");
+    });
+  }
+  deleteChats(chatId: Record<string, number>): Promise<any> {
+    return this.http.delete("/", { data: chatId });
+  }
```

Comment:
Стоит привести к одному виду по всему проекту
```suggestion
  }

  createChat(data: CreateChatData): Promise<any> {
    return this.http.post("/", { data }).catch((e) => {
      console.log(e, "catch");
    });
  }

    deleteChats(chatId: Record<string, number>): Promise<any> {
    return this.http.delete("/", { data: chatId });
  }
```
Вообще с отступами конечно лучше читается